### ISSUE TITLE
Ingestion of PDF document(s)

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss02-international-dataspace-station#2]
## User story
1. As a user
2. I want / need to upload PDF documents
3. So that a knowledge graph can be created from the text information

## Acceptance criteria
* If the document is of required .pdf format, upload and store the document.
* If the document is not of the required .pdf format, return an error message stating wrong format and upload cannot be done.
* User should be notified document(s) are uploaded successfully.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
